config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3367,8 +3367,8 @@
 # wysiwyg.twinpeaks=true
 
 # The file browser to use for picking files in ckeditor
-# The only other option is elfinder.
-# DEFAULT: fckeditor
+# The only other option is fckeditor.
+# DEFAULT: elfinder
 # wysiwyg.editor.ckeditor.browser=elfinder
```

Comment:
Should change this line to a valid value other than the default, ie. `fckeditor`